mongodbatlas/fw_data_source_mongodbatlas_alert_configuration.go

```diff
@@ -291,26 +292,26 @@ func computeAlertConfigurationOutput(alert *matlas.AlertConfiguration, definedOu
 	return resultOutputs
 }
 
-func newTFAlertConfigurationDSModel(apiRespConfig *matlas.AlertConfiguration, projectID string) tfAlertConfigurationDSModel {
+func newTFAlertConfigurationDSModel(apiRespConfig *admin.GroupAlertsConfig, projectID string) tfAlertConfigurationDSModel {
 	return tfAlertConfigurationDSModel{
 		ID: types.StringValue(encodeStateID(map[string]string{
-			encodedIDKeyAlertID:   apiRespConfig.ID,
+			encodedIDKeyAlertID:   *apiRespConfig.Id,
 			encodedIDKeyProjectID: projectID,
 		})),
 		ProjectID:             types.StringValue(projectID),
-		AlertConfigurationID:  types.StringValue(apiRespConfig.ID),
-		EventType:             types.StringValue(apiRespConfig.EventTypeName),
-		Created:               types.StringValue(apiRespConfig.Created),
-		Updated:               types.StringValue(apiRespConfig.Updated),
+		AlertConfigurationID:  types.StringValue(*apiRespConfig.Id),
+		EventType:             types.StringValue(*apiRespConfig.EventTypeName),
+		Created:               types.StringValue(util.TimeToString(*apiRespConfig.Created)),
+		Updated:               types.StringValue(util.TimeToString(*apiRespConfig.Updated)),
```

Comment:
```suggestion
		Created:               types.StringPointerValue(util.TimePtrToStringPtr(apiRespConfig.Created)),
		Updated:              types.StringPointerValue(util.TimePtrToStringPtr(apiRespConfig.Updated)),
```
Would favor using `util.TimePtrToStringPtr` just as a safer option to avoid potential nil pointer dereference.

Reply:
thanks, good idea, adding it in a new commit